README.md

```diff
@@ -321,7 +321,7 @@ WMCO / Windows nodes does not work with the following products:
 * [OpenShift Builds](https://docs.openshift.com/container-platform/latest/cicd/builds/understanding-image-builds.html#understanding-image-builds)
 * [OpenShift Pipelines](https://docs.openshift.com/container-platform/latest/cicd/pipelines/understanding-openshift-pipelines.html#understanding-openshift-pipelines)
 * [OpenShift Service Mesh](https://docs.openshift.com/container-platform/latest/service_mesh/v2x/ossm-about.html)
-* [Red Hat Insights cost management](https://docs.redhat.com/en/documentation/cost_management_service/1-latest)
+* [Red Hat Insights cost management](https://docs.openshift.com/en/documentation/cost_management_service/1-latest)
```

Comment:
Link doesnt work.

Reply:
wait a minute, the jira ticket's description is backwards. It's supposed to replace docs.openshift.com with docs.redhat.com